File: recursion/go/fibonacci.go
package main

import (
	"fmt"
)

func fibonnaci(Number int) int {
	if Number == 0 {
		return Number
	}

	if Number == 1 {
		return Number
	}

	return fibonnaci(Number-1) + fibonnaci(Number-2)
}

func main() {
	fmt.Println("Enter a number:")
	var Number int
	_, err := fmt.Scanf("%d", &Number)

	if err == nil {
		fmt.Println(fibonnaci(Number))
	}
}

File: recursion/swift/Fibonacci.swift
var enterANumberToFindItsFibonacciNumber = 8

func fibonacci(n: Int) -> Int {
    
    if n == 0 {
        return 0
    } else if n == 1{
        return 1
    }
    
    return fibonacci(n: n - 1) + fibonacci(n: n - 2)
}

print("Fibonacci number of \(enterANumberToFindItsFibonacciNumber) is \(fibonacci(n: enterANumberToFindItsFibonacciNumber))")
